src/scss/base/reset/p-toast.scss

```
@@ -23,5 +23,16 @@ body .p-toast .p-toast-message {
 
   &.p-toast-message-error {
     background-color: $wharf_status_red !important;
+
+    a,
+    button {
+      margin-right: 0.8em;
```

I usually preffer rem over em as its typically easier to predict.
```suggestion
      margin-right: 0.8rem;
```